### PR TITLE
Update fallout3.asl

### PR DESCRIPTION
**Pull request by Radioactiv-code**
Made it so that the timer is paused during the intro like New Vegas

### CODE REVIEW

File: fallout3.asl [hunk 1]
```diff
@@ -10,5 +18,5 @@ exit
 
 isLoading
 {
-    return current.isLoading;
+    return vars.isLoading;
```
Comment (drtchops) [inline fallout3.asl]: Instead of adding an update block and a var, you can just do `return current.isLoading || !current.introDone;` here

Comment (Radioactiv-code) [reply fallout3.asl]: Ah thank you :)

File: fallout3.asl [hunk 2]
```diff
@@ -2,13 +2,7 @@ state("Fallout3")
...
     bool isLoading : 0xC76CE8;
 	bool introDone : 0xC771D0;
-}
-update
-{
-   vars.isLoading = false;
-   if ((current.isLoading) || (!current.introDone)) {
-        vars.isLoading = true;
-    }
+	int vault : 0xC78E28;
```
Comment (drtchops) [inline fallout3.asl]: What's this for, is it needed?

Comment (Radioactiv-code) [reply fallout3.asl]: My bad, I was testing for an autosplitter and forgot to remove it

**Review by drtchops — APPROVED** [review]

**Pull request merged**